core/src/main/scala/org/bitcoins/core/protocol/transaction/TxUtil.scala

```diff
@@ -19,55 +19,55 @@ import scala.util.{Failure, Success, Try}
 
```

Comment:
Is everything just a refactor in this file?

Reply:
Yup, just made things easier to reason about